src/composeappmanager.cc

```diff
@@ -206,13 +208,15 @@ ComposeAppManager::AppsContainer ComposeAppManager::getAppsToUpdate(const Uptane
     if (app_data.empty()) {
       // new app in Target
       apps_to_update.insert(app_pair);
+      apps_and_reasons.push_back(app_pair.first + " (new app in target)");
```

Comment:
I don't think this case can happen during the target "sync". IIRC, it can happen only during new target installation. If so, then this is redundant, no point in adding it.

Reply:
I would keep adding a reason for all situations, even when they are not supposed to happen for the "app sync" case. This way we could easily spot an unexpected situation from the event alone. Also, if we eventually implement additional details for a "target update to N+1" event, we already have this information at hand.

Reply:
ok